src/components/upvote/view/upvoteView.js

```diff
@@ -156,9 +221,10 @@ class UpvoteView extends Component {
       iconType = 'AntDesign';
     }
 
-    const _percent = `${(sliderValue * 100).toFixed(0)}%`;
+    const _percent = `${downvote ? '-' : ''}${(sliderValue * 100).toFixed(0)}%`;
     const _amount = `$${amount}`;
     const _totalPayout = totalPayout || '0.000';
+    const _sliderColor = downvote ? '#ec8b88' : '#357ce6';
```

Comment:
there is no `sliderColor` underscore not necessary

Reply:
On video Mustafa post, I saw the color, maybe on mobile it is not visible?!